crates/db/src/traits.rs

```diff
@@ -246,27 +246,27 @@ pub trait CheckpointDatabase {
 
 /// NOTE: We might have to merge this with the [`Database`]
 /// A trait encapsulating provider and store traits to interact with the underlying database for
-/// [`BlobEntry`]
+/// [`PayloadEntry`]
 pub trait SequencerDatabase {
-    type BlobDB: BlobDatabase;
+    type L1PayloadDB: L1PayloadDatabase;
 
-    fn blob_db(&self) -> &Arc<Self::BlobDB>;
+    fn payload_db(&self) -> &Arc<Self::L1PayloadDB>;
 }
```

Comment:
Try to remove this if you can.

Reply:
This should be done while doing sequencer specific refactors. Removing this here might pull up other unexpected things.